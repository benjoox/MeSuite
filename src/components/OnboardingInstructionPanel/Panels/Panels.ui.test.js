import React from 'react'
import renderer from 'react-test-renderer'
import { useRouter } from 'next/router'
import Panel, { ACCOUNT_PAGE_ROUTE_NAME } from '.'

jest.mock('next/router')

describe('Panel section ', () => {
    beforeEach(() => {
        useRouter.mockReturnValue({
            pathname: ACCOUNT_PAGE_ROUTE_NAME,
        })
    })
    it('renders correctly', () => {
        const tree = renderer
            .create(<Panel message="this is a message" />)
            .toJSON()
        expect(tree).toMatchSnapshot()
    })
})
