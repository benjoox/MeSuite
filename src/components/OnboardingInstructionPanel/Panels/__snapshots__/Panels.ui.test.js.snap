// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Panel section  renders correctly 1`] = `
<div
  className="justify-content-center column row"
  style={
    Object {
      "margin": "1rem",
    }
  }
>
  <div
    className="jumbotron"
    style={
      Object {
        "textAlign": "center",
      }
    }
  >
    <p
      style={
        Object {
          "height": "90px",
        }
      }
    >
      this is a message
    </p>
    <section>
      <label
        htmlFor="upload-button"
        style={
          Object {
            "border": "solid grey 1px",
            "borderRadius": "3px",
            "cursor": "pointer",
            "padding": "8px 10px",
          }
        }
      >
        Upload
      </label>
      <input
        accept=".csv"
        hidden={true}
        id="upload-button"
        onChange={[Function]}
        type="file"
      />
      
    </section>
  </div>
</div>
`;
